### PR TITLE
profile and simple_profile takes optional parameter sex

### PR DESCRIPTION
**Pull request by navyad**
This one is for issue https://github.com/joke2k/faker/issues/355.

Optional parameter '**sex**' added for` profile() `and `simple_profile()` allowed values are **'F'** and **'M'**

### CODE REVIEW

**Comment by navyad** [comment]
@coveralls  will add tests.

File: faker/providers/profile/__init__.py [hunk 1]
```diff
@@ -32,7 +33,7 @@ def simple_profile(self):
 
         }
 
-    def profile(self, fields=None):
+    def profile(self, sex=None, fields=None):
```
Comment (fcurella) [inline faker/providers/profile/__init__.py]: to maintain backward compatibility with code using the arguments positionally, the `sex` argument should be added _after_ `fields`

**Comment by navyad** [comment]
@fcurella  following is fixed.

> to maintain backward compatibility with code using the arguments positionally, the sex argument should be added after fields

**Comment by fcurella** [comment]
Thanks @navyad ! ✨

**Pull request merged**